docs/guides/admin/docs/configuration/workflow.md

```diff
@@ -262,6 +262,7 @@ The next operation copies the encoded media to the Opencast distribution channel
 
 * The publish-engage operation uses all media tagged as *rss* or *atom* as input.
 
+<!-- _Are the next sections still part of Creating a Custom Workflow? -->
```

Comment:
I think this is reasonable to have here. If not here, then where? A child page?

Reply:
Ah, my mistake! my comment was actually supposed to say `Are the next sections still part of Create a Workflow?`, aka the first section of this page marked with `h2`. i agree that they belong on this page! 
However, it's somewhat unclear how much belongs in the step-by-step, since the sections after `create a workflow` also seem to be part of creating a workflow x) i might suggest renaming the first section `creating a simple workflow`, and/or adding another heading for the sections following it stating smt like `useful workflow additions` then changing the three sections to `h3`. (am personally for just adding the latter)

Reply:
The whole page is a weird mixture between reference and step-by-step guide and probably needs a re-write.

Reply:
Yep, agree.  @peculiarPrince I'd file this as a separate issue in our tracker.

Reply:
Issue opened!